bench/macro/lsm-tree-bench-wp8.hs

```diff
@@ -90,14 +90,30 @@ type B = Void
 instance LSM.Labellable (K, V, B) where
   makeSnapshotLabel _ = "K V B"
 
--- We generate keys by hashing a word64 and adding two "random" bytes.
--- This way we can ensure that keys are distinct.
+-- We generate 34 keys by using a PRNG to extend a word64 to 32 bytes and then
+-- adding two "random" bytes. This way we can ensure that keys are distinct.
```

Comment:
```suggestion
-- We generate 34 byte keys by using a PRNG to extend a word64 to 32 bytes and then
-- adding two "random" bytes. This way we can ensure that keys are distinct.
```

So the last two bytes are not uniformly distributed? But seems to be okay.

Reply:
Oh, and maybe the `TODO 2024-04-29 consider alternative methods of implementing key generation` is not needed any more now?